sdk/instance/codec.go

```diff
@@ -0,0 +1,16 @@
+package instancesdk
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+)
+
+// ModuleCdc is the codec for the module
+var ModuleCdc = codec.New()
```

Comment:
I don't like global variables, but ...

just wonder if we can create one global codec instead of creating many and then juggle them around the source code.

But if this cdc will be used only in for decode msgs then it's fine.

Btw. This one is empty, maybe we don't need init func in that case.

Reply:
Agree, multiple codecs are a mess especially that some are as global variables and some in dependencies. 
I will create a single codec global variable that every package will register to.